### PR TITLE
Fix the broken dryrun tests with Trusty deployment

### PR DESCRIPTION
**Pull request by nehachopra27**

### CODE REVIEW

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot [hunk 1]
```diff
@@ -11,29 +13,31 @@ ${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
 Verify TrustyAI Operator Installation
     [Documentation]    Verifies that the TrustyAI operator has been
     ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
-    [Tags]    Smoke
-    ...       Tier1    ODS-2481
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Deployment
-    Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI ReplicaSets Info
-    Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI Container Names
+    [Tags]    Tier1
```
Comment (jgarciao) [inline ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:16]: tier1 pipeline doesn't exist yet. Can you add `Sanity` too?

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot [hunk 2]
```diff
@@ -11,29 +13,31 @@ ${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
...
+    ...       ODS-2481
+    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Operator Deployment
```
Comment (jgarciao) [inline ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:18]: (Optional improvement): Instead of adding ` Run Keyword And Continue On Failure` on all lines you can add `robot:continue-on-failure` or `robot:recursive-continue-on-failure` in the  Tags section

https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#enabling-continue-on-failure-using-tags

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot (hunk 1 above)
Comment (nehachopra27) [reply ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot]: Done

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot (hunk 2 above)
Comment (nehachopra27) [reply ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot]: Done, Thanks! Learned something new.

**Review by jgarciao — APPROVED** [review]

**Comment by jstourac** [comment]
Since we're touching this code - these robocop warnings about line lengths could be easily avoided in those cases, do you mind to update these?

Also, could you confirm this has been tested with a successful run?

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot [hunk 3]
```diff
@@ -11,29 +13,31 @@ ${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
...
 
 
 *** Keywords ***
-Verify trustyai-service-operator-controller-manager Deployment
-    [Documentation]    Verifies the  deployment of the trustyai operator in the namespace
-    Wait For Pods To Be Ready   label_selector=app.kubernetes.io/created-by=trustyai-service-operator
-    ...    namespace=${APPLICATIONS_NAMESPACE}    exp_replicas=1
+Verify TrustyAI Operator Deployment
+    [Documentation]    Verifies the  deployment of the trustyai operator in the Applications namespace
+    Wait For Pods Number  1
```
Comment (bdattoma) [inline ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:26]: why this change?

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot (hunk 3 above)
Comment (nehachopra27) [reply ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:26]: The Operator name was changed from now and when the PR was written and it was failing. Fixed it now. Works fine.

Comment (bdattoma) [reply ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:26]: ok but why changing the kw?

Comment (nehachopra27) [reply ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot:26]: kW title has been changed.
